investor_wallet_platform_base/models/subscription_request.py

```diff
@@ -57,6 +57,7 @@ def create(self, vals):
         subscription_request.send_new_subscription_request_notification_email(
             is_company=False
         )
+        subscription_request._compute_display_national_number()
```

Comment:
Are you shure that this compute method should be called manually at creation of the object ?